book/special.qmd

```diff
@@ -878,27 +751,31 @@ x1 = mean(m$lon)
 x2 = mean(m$lat)
 m = map_data("world")
 m = m %>% dplyr::filter(long > -15, long < 40, lat > 36, lat <  60)
-m$dist = -sqrt((m$lon - x1)^2 + (m$lat - x2)^2)
-m$rand = rnorm(nrow(m), mean(m$dist), sd(m$dist))
+m$dist = (-sqrt((m$lon - x1)^2 + (m$lat - x2)^2)) * 10e5
+if (min(m$dist) < 0) {
+  m$dist = m$dist - min(m$dist)
+}
+m$rand = runif(nrow(m), min(m$dist), max(m$dist))
 cou = m %>% group_by(region) %>% summarize(long = mean(long), lat = mean(lat))
 
 p = ggplot(data = m) +
-  scale_fill_gradient2(low = "white", mid = "orange", high = "red", midpoint = mean(m$dist)) +
+  scale_fill_gradient2(low = "white", mid = "lightpink", high = "black", midpoint = mean(m$dist)) +
   theme_minimal() +
-  theme(legend.position = "n") +
   xlim(-10, 25) + ylim(37, 58)
 
 t = geom_text(aes(label = region, x = long, y = lat), data = cou, size = 2)
 
-p1 = p + geom_map(map = m, aes(long, lat, map_id = region, fill = rand)) + t
+p1 = p + geom_map(map = m, aes(long, lat, map_id = region, fill = rand)) + t +
+  theme(legend.position = "n")
 
-p2 = p + geom_map(map = m, aes(long, lat, map_id = region, fill = dist)) + t
+p2 = p + geom_map(map = m, aes(long, lat, map_id = region, fill = dist)) + t +
+  labs(fill = "Cases")
 
-p1 / p2
+p1 / p2 & patchwork::plot_layout(guides = "collect")
 ```
 
-Unlike other tasks we have looked at in this chapter, there is no underlying difference to the implemented learners or measures, instead we provide new resampling methods in `r mlr3spatiotempcv` to account for any overconfidence in predictions that might occur from autocorrelation.
-To make use of these resamplings we have also implemented special tasks in `r mlr3spatial` to hold spatial data.
+Unlike other tasks we have looked at in this chapter, there is no underlying difference to the implemented learners or measures.
+Instead we provide new resampling methods in `r mlr3spatiotempcv` to account for the similarity in the train and test sets during resampling that originates from spatial autocorrelation.
```

Comment:
```suggestion
Instead we provide additional resampling methods in `r mlr3spatiotempcv` to account for the similarity in the train and test sets during resampling that originates from spatiotemporal autocorrelation.
```

The methods are not new, i.e. all of them existed before and we just "use" them. There is no method that has been newly developed in tempcv.

I think adding a disclaimer here regarding the use of the words "spatial autocor." or "spatiotemporal autocor." would be good. I would prefer to use/introduce the acronym STAC (spatiotemporal autocorrelation) and state that it depends on the data and context whether the autocorrelation is spatial, temporal or spatiotemporal.
This way one does not always have to switch between the different terms in each section and it is clear that the focus is on accounting for a combination of all, if existent.

EDIT: I see there is a disclaimer/note coming further down the text. I'd move it up so readers see it earlier and get clarification on the use of the words and different types of autocor.